src/main/java/network/brightspots/rcv/TabulatorSession.java

```diff
@@ -250,6 +251,7 @@ List<String> tabulate(String operatorName, LoadedCvrData expectedCvrData,
             exceptionsEncountered.add(TabulationAbortedException.class.toString());
           } else {
             runTabulationForConfig(config, castVoteRecords.getCvrs(), progress);
+            castVoteRecords.printSummary();
```

Comment:
Curious why this is part of this PR? Just something you wanted to add opportunistically? Or is it some weird artifact from a merge?

Reply:
Nevermind, just saw your note in #821!